### ISSUE TITLE
how to determine page breaks on original docx file javascript

### ISSUE THREAD

**Issue by kertom** [mwilliamson/mammoth.js#381]
Hi, i am using mammoth.js in order to extract text from a word document and convert it to html. Is there a way to get the page breaks that exist on the original document ,so that i know which contents belong to each page/ the page distinction? I was wondering if you could help me, please?
Thank you in advance.

Best regards,
Thomas

**Comment by mwilliamson** [comment]
No, I'm afraid there's currently no way to get the pages from the original document.
